06-building-a-program-start-to-finish/delivering-your-product/README.md

```diff
@@ -0,0 +1,9 @@
+#### Output
+- **Using**: Medium
+- **Deliver**: a link to a post detailing your product and sharing your learning events/experiences
```

Comment:
@q-ode This is great to target the skills you have outlined, however do you think it'll be more useful if we don't stifle creativity by mandating they should blog about the outputs, what if there is some other problem they'd like to talk about?